app/src/main/java/org/oppia/android/app/options/OptionsActivity.kt

```diff
@@ -13,7 +13,7 @@ private const val SELECTED_OPTIONS_TITLE_KEY = "SELECTED_OPTIONS_TITLE_KEY"
 private const val SELECTED_FRAGMENT_KEY = "SELECTED_FRAGMENT_KEY"
 const val READING_TEXT_SIZE_FRAGMENT = "READING_TEXT_SIZE_FRAGMENT"
 const val APP_LANGUAGE_FRAGMENT = "APP_LANGUAGE_FRAGMENT"
-const val DEFAULT_AUDIO_FRAGMENT = "DEFAULT_AUDIO_FRAGMENT"
+const val AUDIO_LANGUAGE_FRAGMENT = "DEFAULT_AUDIO_FRAGMENT"
```

Comment:
Just update the value too. `"DEFAULT_AUDIO_FRAGMENT"` to `"AUDIO_LANGUAGE_FRAGMENT"`